Libraries/Modal/Modal.js

```diff
@@ -138,6 +153,18 @@ class Modal extends React.Component {
   static contextTypes = {
     rootTag: PropTypes.number,
   };
+  constructor(props: Object) {
+    super(props);
+    Modal._confirmProps(props);
+  }
+  componentWillReceiveProps(nextProps: Object) {
```

Comment:
Could you send me a link to a code style spec document? I had been assuming that `npm run lint` and @eslint-bot had been the declarative code style requirements. If not, a link would be helpful both now, and committed in the `CONTRIBUTING.md` file.